cli/install/install.go

```diff
@@ -110,6 +110,9 @@ type InstallCtx struct {
 	IncDir string
 	// Install development build.
 	DevBuild bool
+	// skipMasterUpdate is set if user doesn't want to check for latest master
+	// version and update for it if master version already exists.
+	skipMasterUpdate bool
```

Comment:
Isn't it the default when I run in `no-prompt` mode?

Reply:
It is just inherit no-prompt flag from global context into `InstallCtx`.

Reply:
So, why do we need multiple flags making the same thing?

Reply:
Because we don't have global context in `install`'s functions scope. I haven't found anything else but to inherit from global.